doc/source/set_operations.rst

```diff
@@ -19,58 +19,99 @@ The basic idea is demonstrated by the graphic below but keep in mind that overla
+    from shapely.geometry import Polygon
+    polys1 = gpd.GeoSeries([Polygon([(0,0), (2,0), (2,2), (0,2)]),
+                            Polygon([(2,2), (4,2), (4,4), (2,4)])])
+    polys2 = gpd.GeoSeries([Polygon([(1,1), (3,1), (3,3), (1,3)]),
+                            Polygon([(3,3), (5,3), (5,5), (3,5)])])
 
-    # Select some columns
-    countries = world[['geometry', 'name']]
+    df1 = gpd.GeoDataFrame({'geometry': polys1, 'df1':[1,2]})
+    df2 = gpd.GeoDataFrame({'geometry': polys2, 'df2':[1,2]})
 
-    # Project to crs that uses meters as distance measure
-    countries = countries.to_crs('+init=epsg:3395')[countries.name!="Antarctica"]
-    capitals = capitals.to_crs('+init=epsg:3395')
+These two GeoDataFrames have some overlapping areas:
 
-To illustrate the ``overlay`` function, consider the following case in which one wishes to identify the "core" portion of each country -- defined as areas within 500km of a capital -- using a ``GeoDataFrame`` of countries and a ``GeoDataFrame`` of capitals.
+.. ipython:: python
+
+    ax = df1.plot(color='red');
+    @savefig overlay_example.png width=5in
+    df2.plot(ax=ax, color='green');
+
+
+We illustrate the different overlay modes with the above example. First, with
+``how='intersection'``, it returns the geometries shared by both GeoDataFrames:
+
+.. ipython:: python
+
+    res_intersection = gpd.overlay(df1, df2, how='intersection')
+    res_intersection
+
+    ax = res_intersection.plot()
+    df1.plot(ax=ax, facecolor='none');
+    @savefig overlay_example_intersection.png width=5in
+    df2.plot(ax=ax, facecolor='none');
+
+Using ``how='union'``, all geometries from combining both GeoDataFrames are
+returned:
 
 .. ipython:: python
 
-    # Look at countries:
-    @savefig world_basic.png width=5in
-    countries.plot();
+    res_union = gpd.overlay(df1, df2, how='union')
+    res_union
+
+    ax = res_union.plot()
+    df1.plot(ax=ax, facecolor='none');
+    @savefig overlay_example_union.png width=5in
+    df2.plot(ax=ax, facecolor='none');
+
+To obtain the geometries that are part of ``df1`` but are not contained in
+``df2``, you can use ``how='difference'``:
+
+.. ipython:: python
 
-    # Now buffer cities to find area within 500km.
-    # Check CRS -- World Mercator, units of meters.
-    capitals.crs
+    res_difference = gpd.overlay(df1, df2, how='difference')
+    res_difference
 
-    # make 500km buffer
-    capitals['geometry']= capitals.buffer(500000)
-    @savefig capital_buffers.png width=5in
-    capitals.plot();
+    ax = res_difference.plot()
+    df1.plot(ax=ax, facecolor='none');
+    @savefig overlay_example_difference.png width=5in
+    df2.plot(ax=ax, facecolor='none');
 
 
-To select only the portion of countries within 500km of a capital, we specify the ``how`` option to be "intersect", which creates a new set of polygons where these two layers overlap:
+``how='symmetric_difference'`` is the opposite of ``'intersection'`` and returns
```

Comment:
I'm move up to under intersection. Since you're defining them in relation to one another, makes sense they be neighbors.